.env.example

```
@@ -104,9 +104,13 @@ REGULATORY_POLITE_DELAY=2         # Delay between scraper requests (seconds)
 # -----------------------------------------------------------------------------
 # Supabase (Auth + realtime DB)
 # -----------------------------------------------------------------------------
+# Frontend (Next.js) uses the NEXT_PUBLIC_ prefix; backend services use the plain names.
+# Set ALL FOUR so both frontend and backend services pick up the correct values.
 NEXT_PUBLIC_SUPABASE_URL=https://YOUR_PROJECT_REF.supabase.co
+SUPABASE_URL=https://YOUR_PROJECT_REF.supabase.co          # Same value — used by backend services (admin, etc.)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)